source/ore/routes/deploy-version.rst

```diff
@@ -0,0 +1,57 @@
+========================
+Deploy Project Version
+========================
+
+**POST /api/v1/projects/:pluginId/versions/:version**
+
+Deploys a new version for the specified project. The body should be sent as ``multipart/form-data`` with the fields shown below. Returns a JSON representation of the deployed version.
+
++-------------+------------------------------+------------------------------------------------------------------------------------------------+
+| Name        | Data Type                    | Description                                                                                    |
++=============+==============================+================================================================================================+
+| apiKey      | String                       | An Ore deployment key gotten through the Ore settings.                                         |
++-------------+------------------------------+------------------------------------------------------------------------------------------------+
+| channel     | String                       | The channel the version should be in.                                                          |
++-------------+------------------------------+------------------------------------------------------------------------------------------------+
+| recommended | Boolean                      | If this version should be set as the recommended version. Defaults to true.                    |
++-------------+------------------------------+------------------------------------------------------------------------------------------------+
+| forumPost   | Boolean                      | Overrides the project setting of if a forum post should be created for this version. Optional. |
```

Comment:
Does this really mean "If True, create a forum post for this version. This overrides the project setting." ?

Reply:
Maybe we could phrase it like this. 
`Whether a forum post should be created or not. If no value is specified, it will default to the project's setting.`